cves/CVE-2012-0848.yml

```diff
@@ -344,7 +362,12 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
```

Comment:
Maybe also a lack of testing could me a mistake, since you mentioned that there were no unit tests involved with this vulnerability.

Reply:
Identifying the cause of the mistake is different from identifying that a mistake exists.